CHANGELOG.md

```diff
@@ -1,3 +1,80 @@
+## v0.21.0 (2023-10-18)
+
+## What's Changed
+
+Release highlights:
+* Implemented a req/resp protocol that provides information about the node's medatadata
```

Comment:
perhaps add reference to
https://rfc.vac.dev/spec/66/

Reply:
Great idea! Just added the link to it. Thanks!